### PR TITLE
[N/A] feat(travis): add travis config and deploy script

### PR DESCRIPTION
**Pull request by ksafranski**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-
-machine:
-  node:
-    version: 6
-
-test:
-  pre:
-    - npm run lint
-  post:
-    - bash <(curl -s https://codecov.io/bash)
```
Comment (levithomason) [inline circle.yml:14]: This got lost

**Review by levithomason — CHANGES_REQUESTED** [review]

File: circle.yml (hunk 1 above)
Comment (ksafranski) [reply circle.yml:14]: Fixed

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
```
Comment (levithomason) [inline deploy.sh:2]: We need `set -e` in here, else, this will run all steps even on a failure.

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
I think this is good to go now.

**Comment by levithomason** [comment]
Good times

**Comment by dvdzkwsk** [comment]
image

**Comment by levithomason** [comment]
2018

**Comment by kyleturco** [comment]
Haha, yesterday was old PR closing day. Always a fun day.

**Comment by ksafranski** [comment]
Wow

**Comment by dvdzkwsk** [comment]
Oh, hey @Fluidbyte . How's it going?

**Comment by ksafranski** [comment]
Not bad, miss you guys.

**Comment by dvdzkwsk** [comment]
Same. We should keep this thread open forever.

**Pull request closed without merging**